app/kubernetes/components/datatables/configurations-datatable/configurationsDatatable.html

```diff
@@ -90,7 +90,7 @@
         </div>
         <div ng-if="$ctrl.isAdmin && !$ctrl.settings.showSystem" class="flex flex-row w-full">
           <span class="small text-muted mt-1 vertical-center">
-            <pr-icon icon="'info'" feather="true" mode="'primary'" class="vertical-center"></pr-icon>
+            <pr-icon icon="'info'" mode="'primary'" class="vertical-center"></pr-icon>
```

Comment:
EE doesn't have the class prop. I guess it's not needed

https://github.com/portainer/portainer-ee/pull/2472/files#diff-06d7328e875e75f0fd2e75c8f4802cf7df33c0c9936a5456b5bdf4736cc15bc3R93

Reply:
not needed now the Icon.tsx styles are updated - removed from CE